generated/nidaqmx/nidaqmx_library.cpp

```diff
@@ -53,7 +54,7 @@ int32 NiDAQmxLibrary::ClearTask(TaskHandle task)
 #endif
 }
 
-int32 NiDAQmxLibrary::CreateAIVoltageChan(TaskHandle task, const char physicalChannel[], const char nameToAssignToChannel[], int terminalConfig, float64 minVal, float64 maxVal, int units, const char customScaleName[])
+int32 NiDAQmxLibrary::CreateAIVoltageChan(TaskHandle task, const char physicalChannel[], const char nameToAssignToChannel[], int32 terminalConfig, float64 minVal, float64 maxVal, int32 units, const char customScaleName[])
```

Comment:
Note: the CppHeaderParser tool that I'm using has a quirk where the `int32` typedef reports it's type as the underlying type `int` (but other similar ones do not). This started to cause a problem with out params. I fixed the metadata generation to correct this across the board, so it's in all the diffs.